core/db/migrate/20161129035810_add_index_to_spree_payments_number.rb

```diff
@@ -0,0 +1,5 @@
+class AddIndexToSpreePaymentsNumber < ActiveRecord::Migration
+  def change
+    add_index 'spree_payments', ['number'], unique: true
```

Comment:
@alexstoick yeah we'll do those in eagle (e.g. like [this](https://github.com/Lostmyname/eagle/blob/master/db/migrate/20161121181653_add_index_to_referrals_code.rb)).  that's not fit for solidus itself though because it only works on postgres.